app/login/page.jsx

```diff
@@ -1,9 +1,30 @@
 import Link from 'next/link'
+import Logo from '@/components/Logo'
 
 export default function Login() {
   return (
     <main className="flex min-h-screen flex-col items-center">
-      <div>HelpRat Login</div>
+      <Logo />
+      <form>
+        <div className="relative mb-4">
```

Comment:
i would put a div around the all of the input fields for easier styling, but if you have a plan - go for it